.toys/.data/releases.yml

```diff
@@ -114,6 +114,10 @@ gems:
     directory: instrumentation/restclient
     version_constant: [OpenTelemetry, Instrumentation, RestClient, VERSION]
 
+  - name: opentelemetry-instrumentation-ruby_kafka
```

Comment:
Why not just name this `opentelemetry-instrumentation-kafka` (no `ruby_`)

Otherwise we should do ruby_elasticsearch, ruby_on_rails, mongo_ruby_driver, etc.

Reply:
This instrumentation is for https://rubygems.org/gems/ruby-kafka/versions/1.3.0

Not to be confused with https://rubygems.org/gems/kafka/versions/0.5.2

I don't like this naming either, but I'm attempting to prevent future ambiguity.

Reply:
d'oh fair point! thanks